scripts/split-paired-reads.py

```diff
@@ -78,39 +97,28 @@ def main():
         out1 = os.path.basename(infile) + '.1'
         out2 = os.path.basename(infile) + '.2'
 
+    # OVERRIDE defaults with -1, -2
+    if args.output_first:
+        out1 = args.output_first
+    if args.output_second:
+        out2 = args.output_second
+
     fp_out1 = open(out1, 'w')
     fp_out2 = open(out2, 'w')
 
-    # is input file FASTQ or FASTA? Determine.
-    is_fastq = False
-    record = iter(screed.open(infile)).next()
-
-    if hasattr(record, 'accuracy'):
-        is_fastq = True
-
     counter1 = 0
     counter2 = 0
     index = None
     for index, record in enumerate(screed.open(infile)):
-        if index % 100000 == 0:
+        if index % 100000 == 0 and index:
```

Comment:
On Wed, Feb 11, 2015 at 02:50:55AM -0800, Michael R. Crusoe wrote:
> >      counter1 = 0
> >      counter2 = 0
> >      index = None
> >      for index, record in enumerate(screed.open(infile)):
> > -        if index % 100000 == 0:
> > +        if index % 100000 == 0 and index:
> 
> Why 'and index'? 

I hate seeing the '0' output.